elephant_vending_machine/views.py

```diff
@@ -38,27 +42,74 @@ def run_trial():
         response = 'No trial_name specified'
     return response
 
+def allowed_file(filename):
+    """Determines whether an uploaded image file has an allowed extension.
+
+    Returns:
+        True if filename includes extension and extension is an allowed extension
+        False otherwise
+    """
+    return '.' in filename and \
+           filename.rsplit('.', 1)[1].lower() in ALLOWED_EXTENSIONS
+
 
-@APP.route('/add-image', methods=['POST'])
+@APP.route('/image', methods=['POST'])
 def upload_image():
-    """Responds with 'File received' string
+    """Return string indicating result of request
+
+    **Example request**:
+
+    .. sourcecode:: http
+      
+      POST /image HTTP/1.1
+      Host: 127.0.0.1:5000
+      Content-Type: multipart/form-data; boundary=--------------------------827430006917349763475527
+      Accept-Encoding: gzip, deflate, br
+      Content-Length: 737067
+      Connection: keep-alive
+      ----------------------------827430006917349763475527
+      Content-Disposition: form-data; name="file"; filename="elephant.jpeg"
+
+      <elephant.jpeg>
+      ----------------------------827430006917349763475527--
+
+    **Example response**:
+
+    .. sourcecode:: http
```

Comment:
```suggestion
    .. sourcecode::
```

Same as above, remove the `http` to stop sphinx from throwing an error even though it builds OK.

Reply:
Never mind, this one is fine